fast_neural_style/net.py

```diff
@@ -0,0 +1,165 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+class VGG(nn.Module):
```

Comment:
We can change ```models.vgg16feature()``` to```torchvision.models.vgg16()``` to convert a torch vgg16 model to a pytorch version. In this example, we have to use ```models.vgg16feature()``` for exporting  4 features ```relu1_2,relu2_2,relu3_3,relu4_3```.
```python
def download_convert_vgg16_model():
       ...
        vgg = models.vgg16feature()
        for src,dst in zip(vgglua.parameters()[0],vgg.parameters()):
            dst[:] = src[:]
        torch.save(vgg.state_dict(), 'vgg16feature.pth')
```